### ISSUE TITLE
dat-node and node-dat-archive

### ISSUE THREAD

**Issue by aaronshaf** [beakerbrowser/dat-node#2]
I am building some node services with Dat. Should go ahead and use this (even though not published on npm yet) or [node-dat-archive](https://github.com/beakerbrowser/node-dat-archive)? If I switch over later, will the local dat folders be interoperable? Thanks!

**Comment by pfrazee** [comment]
> If I switch over later, will the local dat folders be interoperable? 

Not yet. We do want to add support for the other storage mechanism (see https://github.com/beakerbrowser/dat-node/issues/1) and make the two 100% compatible.

> Should go ahead and use this (even though not published on npm yet) or node-dat-archive?

This module is already a much better experience, see https://github.com/pfrazee/node-http-dat-forum and https://github.com/beakerbrowser/dat-workshop-server for examples. I think you'll enjoy working with it more.

The most important missing thing is tests and probably second is the local dat folder interop. And there may be some API changes. PRs would definitely be welcome on any of that, since I'm a bit loaded down elsewhere right now.

Basically, if there's not a feature missing that you 100% need, I'd say go ahead and use this.

**Comment by aaronshaf** [comment]
@pfrazee How are you installing `@beaker/dat-node`? `npm link`?

---

Nevermind, I see elsewhere:

```
    "@beaker/dat-node": "github:beakerbrowser/dat-node",
```

**Comment by pfrazee** [comment]
Yep

**Comment by aaronshaf** [comment]
Btw, THANKS for all you do!

**Comment by pfrazee** [comment]
Oh of course! I hope I didn't come off as complaining- I just wish I could triple my bandwidth. I'm very excited to explore dat outside of the browser as a bigger part of app architectures.
